Description: Disable downloading of Nikto database.
 The Nikto database contains a notice that forbids use of the file with
 anything other than the full Nikto package. Also, downloading at run-time
 would cause problems with writing the file because the config directory
 isn't writeable.
Author: Arthur de Jong <adejong@debian.org>
Forwarded: not-needed

--- a/wapitiCore/attack/mod_nikto.py
+++ b/wapitiCore/attack/mod_nikto.py
@@ -74,6 +74,10 @@ class mod_nikto(Attack):
                 reader = csv.reader(fd)
                 self.nikto_db = [line for line in reader if line != [] and line[0].isdigit()]
         except IOError:
+            # Disable downloading of Nikto database because the license of the file
+            # forbids it.
+            self.nikto_db = []
+            return
             try:
                 print(_("Problem with local nikto database."))
                 print(_("Downloading from the web..."))
